styles/light/confirmSuggest.mcss

```diff
@@ -0,0 +1,28 @@
+ConfirmSuggest {
+  position: fixed
+  border: 1px solid #ddd
+  z-index: 100
+  background: white
+
+  ul {
+    margin: 0
+    padding: 0
+    list-style: none
+    li {
+      padding: 4px 8px
+      font-size: 85%
+      border-bottom: 1px solid #ddd
+      selected {
+        color: #fff
+        background-color: #428bca
+        border-color: darken(#428bca, 5%)
+      }
+      :last-child {
+        border: 0
+      }
+      img {
+        height: 20px
+      }
+    }
+  }
+}
```

Comment:
I think this may have been accidentally, do you think this can/should  be deleted? The same file was included with the "standard" hyphenated filename in https://github.com/ssbc/patchwork/commit/9f9a4c5c0cb02112b8ed10ddfcd2ebe06c052ef5, so I think `confirmSuggest.mcss` can be safely deleted.

Reply:
@christianbundy nice catch! I thought I deleted this and `editTags.mcss` I will remove them.

Reply:
No problem! I was curious what the diff from your branch would look like for #850, but I got a few "file not found" errors and noticed the change. The results are [here](https://github.com/ssbc/patchwork/commit/50b43020b1d5b043166adff255149c690420789e), if you want to double-check your dark theme work. Looks great!